integration-cli/docker_api_containers_test.go

```diff
@@ -370,6 +370,108 @@ func TestBuildApiDockerfilePath(t *testing.T) {
 	logDone("container REST API - check build w/bad Dockerfile path")
 }
 
+func TestBuildApiDockerFileRemote(t *testing.T) {
+	server, err := fakeStorage(map[string]string{
+		"testD": `FROM busybox
+RUN echo from testD
```

Comment:
Actually, I think the test is incorrect - its testing to see if a 'baz' file is created but that file shouldn't be. Its a left-over thought from a previous iteration of this PR.  I'll fix that...